utilities/operator.py

```
@@ -622,3 +623,42 @@ def wait_for_cluster_operator_stabilize(admin_client, wait_timeout=TIMEOUT_20MIN
+        UpgradeStreams.Y_STREAM, or UpgradeStreams.Z_STREAM.
+
+    Raises:
+        ValueError: If the target version is not newer than the current version,
+            or if the stream cannot be determined.
+    """
+    current_cnv_version = parse(version=current_version.split("-")[0])
+    target_cnv_version = parse(version=target_version.split("-")[0])
+
+    if current_cnv_version.major < target_cnv_version.major:
+        return UpgradeStreams.X_STREAM
+    elif current_cnv_version.minor < target_cnv_version.minor:
```

Ruff (0.15.21)

[warning] Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)